lib/arjdbc/postgresql/base/pgconn.rb

```diff
@@ -1,8 +1,8 @@
-module ActiveRecord::ConnectionAdapters::PostgreSQL::OID
-  class PGconn # emulate PGconn#unescape_bytea due #652
```

Comment:
@kares I know it sounds like a lot of work but we have had some level of commitment from people working on Rails to help make our integration less painful.  So perhaps we can at least scope out how AR could do things in a way where we don't emulate/fake-out PG and then get them to commit to our plan?

Also do you happen to know which gems offhand?  I am curious if perhaps we can just PR their checks in a reasonable way where we don't trip over them.

  My main desire is to try and reduce the amount of code within ARJDBC to minimize how much we need to update in the future.  In the case of sqlite3 I think we can eliminate 95% of the Ruby we have with a couple of changes (on rails5).  For postgresql it looks much less possible but I guess I am hoping for some upstream AR changes to help eventually reduce that.

My real desire is I wish Rails AR would just use sequel for all adapters.  We would not even need to have this project :|